rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -810,6 +811,15 @@ def _get_transformer_dimensions(
         if isinstance(transformer_units, dict):
             transformer_units = transformer_units[attribute]
         if transformer_layers > 0 and (not transformer_units or transformer_units < 1):
+            rasa.shared.utils.io.raise_warning(
```

Comment:
`_get_transformer_dimensions` can be called by three components `ResponseSelector`, `DIETClassifier` and `TEDPolicy`. Since we don't know which one is calling (although correction would only happen inside `ResponseSelector` because others have positive `TRANSFORMER_SIZE` by default) we should change the warning message.